Stratis.SmartContracts/UInt256.cs

```diff
@@ -157,39 +157,44 @@ public UInt256(byte[] vch) : this(vch, true)
             return new UInt256(value);
         }
 
-        public static implicit operator UInt256(BigInteger value)
```

Comment:
This can stay as internal if it will simplify some code

Reply:
I thought the same but unfortunately "internal" is not accepted here.

Reply:
you can convert it to a internal public method actually.

`internal static ToUInt256(BigInteger value)`